test/integration/ddl_sharding_key_test.lua

```diff
@@ -569,3 +573,81 @@ pgroup.test_update_cache = function(g)
     sharding_key_as_index_obj = helpers.update_cache(g.cluster, space_name)
     t.assert_equals(sharding_key_as_index_obj, {parts = {{fieldno = 3}}})
 end
+
+pgroup.test_jsonpath_insert = function(g)
+    t.skip('JSONpath is unsupported, see issue #219')
+
+    local space_name = 'customers_jsonpath_key'
+    local customers = helpers.insert_objects(g, space_name, {
+        {
+            id = {customer_id = {unsigned = 1}},
+            name = 'Viktor Pelevin',
+            age = 58,
+            data = {customer = {weight = 82}},
+        },
+        {
+            id = {customer_id = {unsigned = 2}},
+            name = 'Isaac Asimov',
+            age = 72,
+            data = {customer = {weight = 70}},
+        },
+        {
+            id = {customer_id = {unsigned = 3}},
+            name = 'Aleksandr Solzhenitsyn',
+            age = 89,
+            data = {customer = {weight = 78}},
+        },
+        {
+            id = {customer_id = {unsigned = 4}},
+            name = 'James Joyce',
+            age = 59,
+            data = {customer = {weight = 82}},
+        },
+        {
+            id = {customer_id = {unsigned = 5}},
+            name = 'Oscar Wilde',
+            age = 46,
+            data = {customer = {weight = 79}},
+        },
+    })
+    t.assert_equals(#customers, 5)
+end
+
+pgroup.test_jsonpath_delete = function(g)
+    t.skip('JSONpath is unsupported, see issue #219')
+
+    local space_name = 'customers_jsonpath_key'
+    local customers = helpers.insert_objects(g, space_name, {
```

Comment:
Here we don't delete anything.

Reply:
It is an incomplete test for jsonpath as sharding key definition support.
Test has been moved to branch `ligurio/gh-219-jsonpath-sharding-key`.